testing/trino-testing-containers/src/main/java/io/trino/testing/containers/junit/ReportLeakedContainers.java

```diff
@@ -58,6 +59,21 @@ public void testPlanExecutionFinished(TestPlan testPlan)
                 log.info("ReportLeakedContainers disabled");
                 return;
             }
+
+            try {
+                if (ignoredIds.isEmpty()) {
```

Comment:
why?

Reply:
if `ignoredIds` has anything, that means the containers were used and check should run (shouldn't skip)